.github/workflows/linux.yml

```
@@ -39,19 +40,35 @@ jobs:
+      - name: Install Redis
+        run: |
+          sudo apt update
+          sudo apt-get install -y lsb-release curl gpg libnss3-tools
+          curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg
+          sudo chmod 644 /usr/share/keyrings/redis-archive-keyring.gpg
+          echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list
+          sudo apt-get update
+          sudo apt-get install redis
+
+      - name: Start Redis with TLS
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:10:7: Quote this to prevent word splitting

(shellcheck)